script/set-version

```diff
@@ -1,30 +1,30 @@
-#!/usr/bin/env bash
+#!/bin/sh -eux
 
 set -euo pipefail
 
-pvp="$1"
-semantic="$2"
-
-if [ -z "$pvp" ] || [ -z "$semantic" ]; then
+if [ $# -ne 2 ]; then
     echo "Usage: bump-version <VERSION> <DEPENDENCY>"
     exit 1
 fi
 
+pvp="$1"
+semantic="$2"
+
 specs=(
  amazonka/amazonka.cabal
  core/amazonka-core.cabal
  examples/amazonka-examples.cabal
  gen/amazonka-gen.cabal
  test/amazonka-test.cabal
- amazonka-s3-encryption/amazonka-s3-encryption.cabal
```

Comment:
Any reason to remove `amazonka-s3-encryption`? It similarly needs to be version bumped and is somewhat confusing, not auto-generated.

Reply:
No reason, I'll add it back in.